src/components/organisms/Header/MobileHeader.tsx

```diff
@@ -11,15 +13,38 @@ import { BASE_URL } from '@/constants/routes';
 import { ILink } from '@/interfaces/ILink';
 import { Link } from '@/navigation';
 
+import styles from './Header.module.scss';
+
 interface Props {
   activeLink: string;
   links: ILink[];
   onClickLink: (url: string) => void;
 }
 
-export const MobileHeader: React.FC<Props> = () => {
+export const MobileHeader: React.FC<Props> = (props) => {
```

Comment:
Extra `React.FC<Props>` and destructuting for props

Reply:
I agree with destructing, but why `React.FC<Props>` is redundant? :)
Or do you prefer to apply type to arguments (like `({ ... }: Props)`?

Reply:
@usavkov-epam 
yes, you are right - ({ ... }: Props)